go/ct/spc/specification.go

```diff
@@ -663,6 +666,10 @@ func getAllRules() []Rule {
 
 		{
 			Name: "jumpi_to_invalid_destination",
+			Parameter: []Parameter{
+				JumpTargetParameter{},
+				JumpTargetParameter{},
```

Comment:
This is wrong, the second parameter is not the jump target, it is a condition for the jump. A good domain for this might be  a [NumericParameter](https://github.com/Fantom-foundation/Tosca/blob/dd9a9f0ee94d2f53198c33a8e79d7afbd8cfae3b/go/ct/rlz/parameters.go#L25C6-L25C22).

Please also fix this in related rules.